CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+-   Add [Mull (divestos.org)](https://divestos.org/pages/our_apps#mull) to supported browsers list for Autofill
```

Comment:
This needs to go in autofill-parser's changelog.

Reply:
Thanks, I see it now. I put it under 1.1.2 with the previous changes but I'm not sure if it should go under a new header 1.1.3 instead. https://github.com/android-password-store/Android-Password-Store/pull/2613/commits/457b116e5776e045cca4344bad5dbe2d84e5f95b

Reply:
It needs to be under a new `### Added` header in the unreleased section, since this change is not part of the 1.1.2 release that's already out.

Reply:
Oops. I understand, I was confused by 1.1.2 not having sections. I have added those too in a second commit. I also modified the message to clarify it isn't a new browser but a new signature being added.